appveyor.yml

```diff
@@ -1,60 +1,21 @@
 ---
-version: "{build}"
-clone_depth: 10
-init:
-  - set PATH=C:\ruby%ruby_version%\bin;C:\Program Files\7-Zip;C:\Program Files\AppVeyor\BuildAgent;C:\Program Files\Git\cmd;C:\Windows\system32
-  - if %ruby_version%==_trunk (
-      appveyor DownloadFile https://ci.appveyor.com/api/projects/MSP-Greg/ruby-loco/artifacts/ruby_trunk.7z -FileName C:\ruby_trunk.7z &
-      7z x C:\ruby_trunk.7z -oC:\ruby_trunk &
-      C:\ruby_trunk\trunk_pkgs.cmd
-    )
 install:
-  # RI DevKit is only installed in Ruby23 and Ruby23-x64 folders
-  # need to install OpenSSL since standard DevKit doesn't contain it
-  - if "%ri_file%"=="x86" (
-      appveyor DownloadFile https://dl.bintray.com/oneclick/OpenKnapsack/x86/openssl-1.0.2j-x86-windows.tar.lzma &
-      7z e openssl-1.0.2j-x86-windows.tar.lzma &
-      7z x -y openssl-1.0.2j-x86-windows.tar -oC:\ruby23\DevKit\mingw &
-      set b_config="--with-ssl-dir=C:/ruby23/DevKit/mingw --with-opt-include=C:/ruby23/DevKit/mingw/include" &
-      set SSL_CERT_FILE=C:/ruby24-x64/ssl/cert.pem
-    )
-  - if "%ri_file%"=="x64" (
-      appveyor DownloadFile https://dl.bintray.com/oneclick/OpenKnapsack/x64/openssl-1.0.2j-x64-windows.tar.lzma &
-      7z e openssl-1.0.2j-x64-windows.tar.lzma &
-      7z x -y openssl-1.0.2j-x64-windows.tar -oC:\ruby23-x64\DevKit\mingw &
-      set b_config="--with-ssl-dir=C:/ruby23-x64/DevKit/mingw --with-opt-include=C:/ruby23-x64/DevKit/mingw/include" &
-      set SSL_CERT_FILE=C:/ruby24-x64/ssl/cert.pem
-    )
-  - ruby --version
-  - gem  --version
-  - bundler --version
-  - bundle install --without documentation --path vendor/%ruby_version%/bundle
+  # download shared script files
+  - ps: >-
+      if ( !(Test-Path -Path ./shared -PathType Container) ) {
+        $uri = 'https://ci.appveyor.com/api/projects/MSP-Greg/av-gem-build-test/artifacts/shared.7z'
+        $7z  = 'C:\Program Files\7-Zip\7z.exe'
+        $fn = "$env:TEMP\shared.7z"
+        (New-Object System.Net.WebClient).DownloadFile($uri, $fn)
+        &$7z x $fn -owin_gem_test 1> $null
+        Remove-Item  -LiteralPath $fn -Force
+        Write-Host "Downloaded shared files" -ForegroundColor Yellow
+      }
+
 build_script:
-  - bundle config build.eventmachine --no-document --env-shebang %b_config%
-  - bundle exec rake -rdevkit compile -- %b_config%
-test_script:
-  - bundle exec rake -rdevkit test
-on_finish:
- - ruby -v
+  - ps: .\win_gem_test\eventmachine.ps1 $env:gem_bits
+
 environment:
-  TESTOPTS: -v --no-show-detail-immediately
-  ri_file: none
   matrix:
-    - ruby_version: "200"
```

Comment:
Without the `ruby_version` matrix, what is the default Ruby that AppVeyor uses?

Reply:
The file win_gem_test/eventmachine.ps1, lines 16-20 defines the lowest ruby version and how trunk (with & without JIT) is handled.

See PR comment.